samples/mtls-serve/mtls-server.yaml

```diff
@@ -0,0 +1,79 @@
+# Copyright Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: mtls-serve
+  labels:
+    app: mtls-serve
+spec:
+  ports:
+    - port: 443
+      targetPort: 8443
+      protocol: TCP
+      name: https
+  selector:
+    app: mtls-serve
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: mtls-serve
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: mtls-serve
+  template:
+    metadata:
+      labels:
+        app: mtls-serve
+    spec:
+      containers:
+        - name: mtls-serve
+          image: localhost:5000/mtls-server:latest
```

Comment:
I bet "serve" and "server" is going to trip someone up one day

Reply:
Good point. We really need to update the manifest to point to our published version if we're going to use this.

Reply:
to be explicit I would recommend you rename "mtls-serve" (directory, command) to "mtls-server"

Reply:
gotcha, took another crack at it :)